src/main.cpp

```diff
@@ -36,6 +37,23 @@ int main(int argc, char *argv[])
     QApplication a(argc, argv);
     a.setQuitOnLastWindowClosed(true);
 
+	{
+        QFile f(":/qdarkstyle/uistyle.qss");
+		if (!f.exists()) {
+			printf("Unable to set stylesheet, file not found\n");
+		}
+		else {
```

Comment:
Don't you think we should make it a preference ?

As far as I understand this overrides the default theme of the operating system. I suspect that some users have dark theme defined at the Windows/Linux level, and it would be nice if we could avoid to enforce it.